process/smartContract/builtInFunctions/esdtTransfer.go

```diff
@@ -76,6 +83,9 @@ func (e *esdtTransfer) ProcessBuiltinFunction(
 	if err != nil {
 		return nil, err
 	}
+	if e.shardCoordinator.ComputeId(vmInput.RecipientAddr) == core.MetachainShardId {
```

Comment:
same here, will this be backwards compatible?

Reply:
it is backward compatible with mainnet. on testnet there was only one transaction like this yesterday night